src/main/java/seedu/address/model/person/Github.java

```diff
@@ -28,7 +28,7 @@ public Github(String githubUsername) {
      * Returns if a given string is a valid email.
```

Comment:
You might want to change the java docs for this method. It still says `valid email` and not `valid Github Username`

Reply:
Good point! I have changed it